ccp/lib/clients/base.py

```diff
@@ -23,3 +25,57 @@ def __init__(self, base_command):
         """
         super(Client, self).__init__()
         self.base_command = base_command
+
+
+class APIClient(Client):
+    """
+    Base class for all clients that query an API Server for information
+    """
+    def __init__(
+            self,
+            server,
+            secure=True,
+            verify_ssl=True,
+            authorization=None
+    ):
+        """
+        Initialize APIClient
+        :param server: The URL/IP server to hit.
```

Comment:
remove the dot (.) for all the strings which are not sentences

Reply:
Sorted